### ISSUE TITLE
ref is not defined

### ISSUE THREAD

**Issue by oiluis** [danielroe/nuxt-time#290]
### 🐛 The bug

Am I the only one who is facing this error ?
Just open the playground and you will see the following error

```
ref is not defined

at _sfc_main.setup (./node_modules/nuxt-time/dist/runtime/components/NuxtTime.js#cjs:112:23)
at callWithErrorHandling (./node_modules/@vue/runtime-core/dist/runtime-core.cjs.js#cjs:200:19)
at setupStatefulComponent (./node_modules/@vue/runtime-core/dist/runtime-core.cjs.js#cjs:7768:25)
at setupComponent (./node_modules/@vue/runtime-core/dist/runtime-core.cjs.js#cjs:7729:36)
at renderComponentVNode (./node_modules/@vue/server-renderer/dist/server-renderer.cjs.js#cjs:645:15)
at Module.ssrRenderComponent (./node_modules/@vue/server-renderer/dist/server-renderer.cjs.js#cjs:86:10)
at _sfc_ssrRender (./pages/index.js#cjs:37:31)
at renderComponentSubTree (./node_modules/@vue/server-renderer/dist/server-renderer.cjs.js#cjs:715:9)
at renderComponentVNode (./node_modules/@vue/server-renderer/dist/server-renderer.cjs.js#cjs:664:12)
```

### 🛠️ To reproduce

https://stackblitz.com/github/danielroe/nuxt-time/tree/main/playground?file=package-lock.json,nuxt.config.ts,pages%2Findex.vue

### 🌈 Expected behaviour

To use the component as designed / expected.

**Comment by danielroe** [comment]
Oops.

**Comment by oiluis** [comment]
@danielroe do you know if is there any workaround to fix it - just asking because I found this component pretty useful ... no pressure at all :)

**Comment by danielroe** [comment]
I will release a new version shortly. For now, just use the previous version of the module.

**Comment by danielroe** [comment]
should be resolved in v1.0.1 🙏
